stories/Molecules/FooterNavigation/FooterIcons/footerIcons.scss

```diff
@@ -6,11 +6,15 @@
 .footer__icons {
   display: flex;
   padding-left: 0;
+  width: 100%;
 
   li {
     list-style: none;
 
     a {
+
+      $socials: ('facebook', 'instagram', 'undp-linkedin', 'twitter', 'youtube');
```

Comment:
@Shubham-Sharma-77 Any specific reason why this needs to be `undp-linkedin`? Otherwise change this to just `linkedin`.